packages/qgrid-ngx/tsconfig.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Why do we need this file?

Reply:
I need it, but the repository does not. Accidental commit.